apollo-federation/src/sources/connect/query_graph.rs

```diff
@@ -190,29 +196,14 @@ fn process_subselection(
     node_cache: &mut IndexMap<Name, NodeIndex<u32>>,
     properties_path: Option<Vec<Property>>,
 ) -> Result<NodeIndex<u32>, FederationError> {
-    // Reference for working with the entry API
```

Comment:
do you have a TODO for using the cache for leaf types?

Reply:
This comment was mostly here as reference. The existing code already uses the cache